cstore_fdw.c

```diff
@@ -358,8 +366,13 @@ CStoreProcessUtility(Node * parseTree, const char *queryString,
 		{
 			truncateStatement->relations = otherTablesList;
 
-			CALL_PREVIOUS_UTILITY(parseTree, queryString, context, paramListInfo,
+			CALL_PREVIOUS_UTILITY((Node *) truncateStatement, queryString, context, paramListInfo,
 								  destReceiver, completionTag);
+                        /* restore the former relation list. Our
+                         * replacement could be freed but still needed
+                         * in a cached plan. A truncate can be cached
+                         * if run from a pl/pgSQL function */
+                        truncateStatement->relations = allTablesList;
```

Comment:
this is not needed when you make a copy of parse tree.

Reply:
Indeed. Fixed in a13ace227f139622724f347b33c063a3d1aa19e5